src/routes/create-dictionary.svelte

```diff
@@ -195,7 +198,9 @@
       <label for="glosses" class="block text-sm font-medium leading-5 text-gray-700">
         {$_('create.gloss_dictionary_in', {
           default: 'Make dictionary available in...',
-        })}*
+        })}* - {$_('create.add_gloss_msg', {
+          default: 'You can add more glossing languages later in under Settings',
+        })}
```

Comment:
@Danble All settings except for url can be changed later in settings so this feels like an unneeded string for the translators to tackle. I removed the code, can you remove from the spreadsheet please?